libclingo/src/control.cc

```diff
@@ -2196,6 +2201,29 @@ extern "C" bool clingo_control_register_observer(clingo_control_t *control, clin
     GRINGO_CLINGO_CATCH;
 }
 
+extern "C" bool clingo_control_register_backend(clingo_control_t *control, clingo_backend_type_t type, char const *file, bool replace) {
+    GRINGO_CLINGO_TRY { 
+        auto out = gringo_make_unique<std::ofstream>(file);
+        auto backend = UBackend{}; //Output::make_backend(out, OutputFormat::reify, false, false);
+        switch (type & 0xFFFFFFFC) {
+            case clingo_backend_type_e::clingo_backend_type_reify: {
+                backend = Output::make_backend(std::move(out), Output::OutputFormat::REIFY, (type & 1) == 1, (type &2) == 2);
```

Comment:
Nitpick: inconsistent spacing

Reply:
I don't even see where the inconsistent spacing is anymore. In another project I switched to 100% relying on clang-format. I should setup clang-format at some point but will leave it out of this PR.